spec/mailers/test_mailer_spec.rb

```diff
@@ -3,7 +3,9 @@
 require 'spec_helper'
 
 RSpec.describe Spree::TestMailer do
+  subject(:mail) { described_class.test_email(order) }
```

Comment:
Although this works, test emails are normally generated for the user that requested them (see `test_email(user)` below). So we don't need to create an order, and can refer to the user directly.

But then.. I would argue we don't need to test white labelling on the test email anyway! As long as it sends an email to the right person, it's a good test. 
`±` So I suggest removing the updates on this file, which will save time/resources on CI tests.

Reply:
Okay, thanks David! I will remove that one. :+1: